### DISCUSSION TITLE
Книга жалоб и предложений

### DISCUSSION THREAD

**Discussion by Cutlet69** [getlantern/lantern-forum-ru#14] [Announcements]
Ничего себе перекур, да? Мы активно занимались улучшением работы приложения и соединения. Пожалуйста, поделитесь впечатлениями от Lantern в последние дни/недели/месяцы. Да, мы понимаем, что панамку нам носить пока опасно, но мы стремимся. Спасибо за ваше терпение!

**Comment by Homarata** [comment]
Здравствуйте. Не могу попасть с помощью Лантерн на заблокированный сайт. И можно ли узнать как сменить страну, через которую заходишь. Никакого навигационного меню на сайте нет. Я знаю только включить впн и выключить. И все.

**Reply by Cutlet69** [reply]
Добрый день! Возможность выбора страны будет добавлена в конце года. Сейчас вы автоматически подключаетесь к ближайшему серверу для лучшей скорости соединения

**Comment by Borevlad** [comment]
strava.com - не пускает, хотя и включено.

**Reply by Cutlet69** [reply]
Будем разбираться

**Comment by SomeLoudBoy** [comment]
Жалоб нет, прекрасная программа! Но было бы невероятно полезно если бы информация с сайта (типа расположения сервера, лимит трафика и когда он будет сброшен) была сразу в боковой панели
![image](https://github.com/user-attachments/assets/e31cb013-c9b6-40cc-a2c7-8888ba130e7d)
![image](https://github.com/user-attachments/assets/8787409d-1fd6-43fa-af63-d11ad1ef426a)
![image](https://github.com/user-attachments/assets/91b86347-340a-4a88-9e32-589bb969af69)

**Reply by Cutlet69** [reply]
Добрый день! Взяли в работу

**Comment by ILDanis** [comment]
Столкнулся с проблемой, установил  программу, запустил-ничего не произошло, и так пару раз. В итоге удалил, а после удаления появился белый фон на пол экрана. Перезагрузка не помогает(
![image](https://github.com/user-attachments/assets/a48b51ab-2acd-4dbc-92d8-1543616b3277)

**Comment by Olegas-111** [comment]
Проблема: Lantern не запускается, "No internet connection detected", "Загрузка конфигурации..." и далее ничего не происходит. - скриншот.
![Screenshot_2024-12-24-09-14-40-686_org getlantern lantern](https://github.com/user-attachments/assets/bc71adc1-86ef-4d52-942f-c37f83c65683)

Lantern v.7.9.9

**Comment by MirVashemuDomu** [comment]
Установил на android  и windows не работает. Оператор Ростелеком.

**Reply by MirVashemuDomu** [reply]
Заработало))) спасибо!

**Comment by MirVashemuDomu** [comment]
На android так и не работает((( Оператор Т2
